src/applications/claims-status/components/claim-overview-tab/ClaimOverviewHeader.jsx

```diff
@@ -1,26 +1,32 @@
 import React from 'react';
-import { Toggler } from '~/platform/utilities/feature-toggles';
+import PropTypes from 'prop-types';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
 
-function ClaimOverviewHeader() {
+import { isDisabilityCompensationClaim } from '../../utils/helpers';
+
+export default function ClaimOverviewHeader({ claimTypeCode }) {
+  const { TOGGLE_NAMES, useToggleValue } = useFeatureToggle();
+  const cstClaimPhasesEnabled = useToggleValue(TOGGLE_NAMES.cstClaimPhases);
+  // When feature flag cstClaimPhases is enabled and claim type code is for a disability
+  // compensation claim we show 8 phases instead of 5 with updated description, link text
+  // and statuses
+  const showEightPhases =
```

Comment:
Should I make this a helper function? I dont see that we have any helper functions where we include a feature flag so not quite sure if I should do this or not. LMK your thoughts!

Reply:
Sure, you just have to make the feature toggle value an argument to the helper function. I would go ahead and make it a helper function

Reply:
Going to leave this as is for now and may change it down the road.